packages/clients/package.json

```
@@ -4,19 +4,27 @@
   "version": "0.0.0",
   "type": "module",
   "scripts": {
+    "lint": "eslint ./src --ext .ts",
```

This shorter command will work in the meantime we reconfigure linting properly.
```suggestion
    "lint": "eslint ./src",
```